test/cluster_SUITE.erl

```diff
@@ -1071,10 +1151,10 @@ start_n_nodes(NamePrefix, Count) ->
                  ct:pal("- ~s", [Name]),
                  start_erlang_node(Name)
              end || I <- lists:seq(1, Count)],
-    ct:pal("Started nodes: ~p", [Nodes]),
+    ct:pal("Started nodes: ~p", [[N || {_P, N} <- Nodes]]),
```

Comment:
`_P` and `N` are swapped compared to what `start_erlang_node/1` returns.

Also, could you please use the same variable name as elsewhere, so `_Peer` and `Node`?